.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-stretch-node
```

Comment:
is there a ruby image without node?

Reply:
Yup! Fixed in 9f2046fb7a371bc2b1f100ce5f4262fc3b9ed508